seatunnel-engine/pom.xml

```
@@ -30,5 +30,6 @@
         <module>seatunnel-engine-common</module>
         <module>seatunnel-engine-server</module>
         <module>seatunnel-engine-core</module>
+        <module>seatunnel-check-point</module>
```

I don't think a module is needed for checkpoint because checkpoint is part of the server;
I think this module can be changed to storage to support multiple storages, store data such as checkpoints, conf files, etc.
```suggestion
        <module>seatunnel-storage</module>
```
And submodules will not be prefixed with checkpoint.

---

I updated to seatunnel-engine-storage, but for sub-module, I still keep this. The current interface is basically designed for checkpoint. If support is needed in the future, we will modify it.